### PR TITLE
fix: mutation conflict

### PR DESCRIPTION
**Pull request by elevatebart**
fixes a conflict in Apollo

Screen Shot 2021-11-09 at 11 20 02 AM

Is the original mutation still necessary?

### CODE REVIEW

**Comment by elevatebart** [comment]
@marktnoonan can you have a look?

File: packages/frontend-shared/src/gql-components/ExternalLink.vue [hunk 1]
```diff
@@ -18,20 +18,16 @@ export default defineComponent({
 
 <script setup lang="ts">
 import BaseLink from '../components/BaseLink.vue'
-
-import {
-  App_OpenExternalDocument,
-} from '../generated/graphql'
-
+import { OpenExternalDocument } from '../generated/graphql'
```
Comment (tgriesser) [inline packages/frontend-shared/src/gql-components/ExternalLink.vue:21]:
```suggestion
import { ExternalLink_OpenExternalDocument } from '../generated/graphql'
```

File: packages/frontend-shared/src/gql-components/ExternalLink.vue [hunk 2]
```diff
@@ -18,20 +18,16 @@ export default defineComponent({
...
 import { gql, useMutation } from '@urql/vue'
 
 gql`
-mutation App_OpenExternal ($url: String!) {
+mutation OpenExternal ($url: String!) {
```
Comment (tgriesser) [inline packages/frontend-shared/src/gql-components/ExternalLink.vue:25]: External
```suggestion
mutation ExternalLink_openExternal ($url: String!) {
```

File: packages/frontend-shared/src/gql-components/ExternalLink.vue [hunk 3]
```diff
@@ -18,20 +18,16 @@ export default defineComponent({
...
   openExternal(url: $url)
 }
 `
 
-const openExternalMutation = useMutation(App_OpenExternalDocument)
+const openExternalMutation = useMutation(OpenExternalDocument)
```
Comment (tgriesser) [inline packages/frontend-shared/src/gql-components/ExternalLink.vue:30]:
```suggestion
const openExternalMutation = useMutation(ExternalLink_OpenExternalDocument)
```

**Review by tgriesser — COMMENTED** [review]
Added some suggestions for consistent naming w/ the rest of the codebase

We've talked about getting a lint rule in place for this, I think it's getting to the point of being necessary to prevent these sort of conflicts & to enforce consistency as we continue to build. 

@estrada9166 anything you'd want to take a look at?

File: packages/frontend-shared/src/gql-components/ExternalLink.vue (hunk 1 above)
Comment (elevatebart) [reply packages/frontend-shared/src/gql-components/ExternalLink.vue]: Done in e8f61b3

File: packages/frontend-shared/src/gql-components/ExternalLink.vue (hunk 3 above)
Comment (elevatebart) [reply packages/frontend-shared/src/gql-components/ExternalLink.vue]: Done in e8f61b3

**Review by tgriesser — APPROVED** [review]

**Pull request merged**